### ISSUE TITLE
Apache Commons codec 1.11 dependency is having a vulnerability due to which Restassured is caught by BlackDuck and other scans. 

### ISSUE THREAD

**Issue by pvchaitu** [rest-assured/rest-assured#1725]
Apache Commons codec 1.11 dependency is having a vulnerability due to which Restassured is caught by BlackDuck and other scans to have vulnerability. 

This is the issue in current latest Restassured library 5.3.1 version. Curious when the next release is coming up and if the Commons-codec dependency is going to be upgraded to latest version?

**Comment by gruenich** [comment]
Apache Commons codec 1.11 is a dependency of Apache httpclient 4.5. It was droppen by the switch to httpclient5. I expect that MR #1719 will fix this issue. @pvchaitu, can you confirm?

**Comment by pvchaitu** [comment]
Yes, HttpClient5 (5.2.1) should take care of the issue and has no vulnerabilities as of today. Please also see if you can bump up below dependencies:
johnzon-mapper to 1.2.21 or later
jackson-mapper-asl to 2.15.2 or later

**Comment by j7an** [comment]
Any update on when HttpClient5 will replace http client 4.5.13 in sub dependency?
